api/go.mod

```diff
@@ -17,17 +17,19 @@ require (
 	github.com/json-iterator/go v1.1.12 // indirect
 	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
 	github.com/modern-go/reflect2 v1.0.2 // indirect
-	github.com/onsi/ginkgo/v2 v2.8.4 // indirect
-	github.com/onsi/gomega v1.27.2 // indirect
+	github.com/onsi/ginkgo/v2 v2.9.1 // indirect
+	github.com/onsi/gomega v1.27.3 // indirect
 	github.com/stretchr/testify v1.8.1 // indirect
-	golang.org/x/net v0.7.0 // indirect
-	golang.org/x/text v0.7.0 // indirect
+	golang.org/x/net v0.8.0 // indirect
+	golang.org/x/text v0.8.0 // indirect
 	gopkg.in/check.v1 v1.0.0-20201130134442-10cb98267c6c // indirect
 	gopkg.in/inf.v0 v0.9.1 // indirect
 	gopkg.in/yaml.v2 v2.4.0 // indirect
-	k8s.io/api v0.26.1 // indirect
+	k8s.io/api v0.26.2 // indirect
 	k8s.io/klog/v2 v2.80.1 // indirect
 	k8s.io/utils v0.0.0-20230115233650-391b47cb4029 // indirect
 	sigs.k8s.io/json v0.0.0-20221116044647-bc3834ca7abd // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 )
+
+replace github.com/openstack-k8s-operators/nova-operator/api => github.com/gibizer/nova-operator/api v0.0.0-20230318182909-7589c2d0c019
```

Comment:
we can remove this :smile:

Reply:
ah, I see your comment now about updating this once the nova-operator PR lands.

Reply:
yepp the nova PR landed so I will respin this to depend directly to the merged version. Then when this PR merges I will propose a bump in the openstack-operator that takes both the recent nova-operator and the recent dataplane-operator version used there so we can move in sync.